tests/functional/spec/features/placeholder/placeholder_metadata.spec.js

```diff
@@ -0,0 +1,18 @@
+import MandatoryRadioPage from "../../../generated_pages/placeholder_metadata/mandatory-radio.page";
+import SubmitPage from "../../../generated_pages/placeholder_metadata/submit.page";
+
+describe("Place holder metadata check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_placeholder_metadata.json");
+  });
+  it("Given when i launch survey, then i see radio option with first option as metadata placeholder (ru_name)", () => {
```

Comment:
Needs a little tweak, probably best to extract the Given as it applies to both tests
"Given I launch the placeholder metadata question" 
"When answering the responding unit question ....